include/string.h

```diff
@@ -3,7 +3,6 @@
 
 #include <stddef.h>
 
-void memcpy(void* dst, void* src, size_t num);
```

Comment:
Why'd this get deleted?

Reply:
Was a duplicate of something, either `Runtime/__mem.h` or `cstring.h`.

Reply:
well I think it belongs here, it's C standard and <cstring> is a C++ thing

Reply:
Then our splits are wrong and it needs to be addressed at the TU level, not the header level.

Reply:
Nah where it's implemented is arbitrary, it just needs to be made available from this header. So just have this header include `<Runtime/__mem.h>` and I think we're good.

Reply:
> So just have this header include `<Runtime/__mem.h>` and I think we're good.

Okay.